specification/app/resource-manager/Microsoft.App/preview/2024-02-02-preview/ContainerAppsPatches.json

```diff
@@ -0,0 +1,563 @@
+          },
+          {
+            "$ref": "#/parameters/ContainerAppsNameParameter"
+          },
+          {
+            "name": "$filter",
+            "in": "query",
+            "description": "The filter to apply on the operation.",
+            "required": false,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "ARM operation completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/PatchCollection"
+            }
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "ContainerAppsPatches_ListByContainerApp_0": {
+            "$ref": "./examples/ContainerAppsPatches_ListByContainerApp.json"
+          }
+        },
+        "x-ms-odata": "#/definitions/ContainerAppsContainerAppsPatchResource"
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/containerApps/{containerAppName}/patches/{patchName}": {
+      "get": {
+        "tags": [
+          "ContainerAppsPatches"
+        ],
+        "operationId": "ContainerAppsPatches_Get",
+        "description": "Get details for specific ContainerAppsPatch by patch name.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ContainerAppsNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ContainerAppsPatchNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Patch information retrieved successfully.",
+            "schema": {
+              "$ref": "#/definitions/ContainerAppsPatchResource"
+            }
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ContainerAppsPatches_Get_0": {
+            "$ref": "./examples/ContainerAppsPatches_Get.json"
+          }
+        }
+      },
+      "delete": {
```

Comment:
why is `put` not supported? how is this resource created?

Reply:
addressed in the other comment